web/turkey/survey/models.py

```diff
@@ -124,10 +142,9 @@ class TaskInteraction(_TaskLinkedModel):
     associated with.
 
     The other consideration: We can see what percentage of people are
-    completing our HITs
+    completing our HITs by calculating (task interactions with linked submitted steps)/(task interactions)
     """
```

Comment:
I did away with the completed flag to allow more flexibility and to avoid the inherent denormalization. So now we simply check what data we have